jira/src/test/java/org/arquillian/extension/governor/jira/ITJiraGovernorTest.java

```diff
@@ -25,6 +25,7 @@
 
 @RunWith(Arquillian.class)
 @RunAsClient
+@Jira("ARQ-831")

```

Comment:
please add another test where jira will be placed in test method just as you deleted here in order to cover both cases